examples/example_notebooks/keras_rl2_example.ipynb

```diff
@@ -0,0 +1,1483 @@
+      " 15697/100000: episode: 31, duration: 0.164s, episode steps:  33, steps per second: 201, episode reward: -2177.567, mean reward: -65.987 [-2000.000, -4.554], mean action: 3.394 [0.000, 6.000],  loss: 9491.576722, mse: 2996.744454, mean_q: -16.614981, mean_eps: 0.702080\n",
+      " 16412/100000: episode: 32, duration: 3.357s, episode steps: 715, steps per second: 213, episode reward: -5376.061, mean reward: -7.519 [-2000.000, -0.050], mean action: 2.485 [0.000, 6.000],  loss: 3060.986376, mse: 1224.476242, mean_q: -18.470690, mean_eps: 0.694974\n",
+      " 17182/100000: episode: 33, duration: 3.606s, episode steps: 770, steps per second: 214, episode reward: -5491.166, mean reward: -7.131 [-2000.000, -2.102], mean action: 3.552 [0.000, 6.000],  loss: 4437.352609, mse: 1796.416306, mean_q: -22.864909, mean_eps: 0.680867\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      " 17767/100000: episode: 34, duration: 2.757s, episode steps: 585, steps per second: 212, episode reward: -4019.910, mean reward: -6.872 [-2000.000, -0.331], mean action: 3.607 [0.000, 6.000],  loss: 3416.142353, mse: 1736.472192, mean_q: -27.094901, mean_eps: 0.667994\n",
+      " 17862/100000: episode: 35, duration: 0.467s, episode steps:  95, steps per second: 203, episode reward: -2378.473, mean reward: -25.037 [-2000.000, -1.629], mean action: 3.579 [0.000, 6.000],  loss: 5661.363447, mse: 2469.991381, mean_q: -28.272834, mean_eps: 0.661534\n",
+      " 17913/100000: episode: 36, duration: 0.254s, episode steps:  51, steps per second: 201, episode reward: -2214.496, mean reward: -43.421 [-2000.000, -2.763], mean action: 3.784 [0.000, 6.000],  loss: 1491.557036, mse: 1321.302298, mean_q: -28.985987, mean_eps: 0.660147\n",
+      " 18107/100000: episode: 37, duration: 0.976s, episode steps: 194, steps per second: 199, episode reward: -2805.934, mean reward: -14.464 [-2000.000, -0.779], mean action: 3.784 [0.000, 6.000],  loss: 3986.904849, mse: 2041.443002, mean_q: -28.866613, mean_eps: 0.657820\n",
+      " 18685/100000: episode: 38, duration: 2.800s, episode steps: 578, steps per second: 206, episode reward: -3404.857, mean reward: -5.891 [-2000.000, -0.169], mean action: 3.702 [0.000, 6.000],  loss: 2529.650000, mse: 1759.634946, mean_q: -31.142986, mean_eps: 0.650485\n",
+      " 18726/100000: episode: 39, duration: 0.201s, episode steps:  41, steps per second: 204, episode reward: -2209.227, mean reward: -53.884 [-2000.000, -3.777], mean action: 3.585 [0.000, 6.000],  loss: 1831.572370, mse: 1556.794414, mean_q: -31.470386, mean_eps: 0.644605\n",
+      " 18812/100000: episode: 40, duration: 0.424s, episode steps:  86, steps per second: 203, episode reward: -2432.605, mean reward: -28.286 [-2000.000, -3.227], mean action: 3.721 [0.000, 6.000],  loss: 5377.043741, mse: 2580.409872, mean_q: -31.463692, mean_eps: 0.643399\n",
+      " 18873/100000: episode: 41, duration: 0.293s, episode steps:  61, ste
```

Comment:
I believe it would be nice to also have a graphical representation of the trained agents -> the above depiction from the early training could otherwise leave the impression that the agent is not performing too well after all.

Reply:
test phase is not plotted although arg "visualize=True" is set.
@pramodcm95 can you help out here? Have you had plots during testing?

Reply:
Ya, I did have plots for both training and testing. Just did a small re-run(with lesser steps) and the plots are being generated. It's strange if it disappears later.